cmd/extensions/main.go

```diff
@@ -200,30 +173,14 @@ func main() {
 	// which is a requirements of Stackdriver, otherwise most of time series would be invalid for Stackdriver
 	metrics.SetReportingPeriod(ctlConf.PrometheusMetrics, ctlConf.Stackdriver)
 
-	// Add metrics controller only if we configure one of metrics exporters
-	if ctlConf.PrometheusMetrics || ctlConf.Stackdriver {
-		rs = append(rs, metrics.NewController(kubeClient, agonesClient, kubeInformerFactory, agonesInformerFactory))
-	}
-
 	server.Handle("/", health)
 
-	gsCounter := gameservers.NewPerNodeCounter(kubeInformerFactory, agonesInformerFactory)
```

Comment:
You'll probably need this as well, when you add back the allocations server.

Reply:
added back.